committee/heightmanager_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package committee
+
+import (
+	"strconv"
+	"testing"
+	"time"
+)
+
+var baseTime = time.Unix(int64(1546272000), 0) // 2019-01-01 00:00:00
+var hm *heightManager
+
+func TestNewHeightManager(t *testing.T) {
+	hm = newHeightManager()
+	if hm == nil {
```

Comment:
require.NotNil(t, hm)

Reply:
got github.com/stretchr/testify/require, nice library, figure it out now.